Additions to account restriction article (#2310)
ppy/osu-wiki: wiki/Help_Centre/Account_Restrictions/en.md

If your account has been restricted, do not panic. Please follow these steps:
- **Wait 24 hours.** Some restrictions are the results of automated processes. As we check every new restriction daily, please wait 24 hours - if your restriction was erroneous, it will hopefully be lifted within that time.
- **Contact our support team.** If your account remains restricted after that time, please send an email to [accounts@ppy.sh](mailto:accounts@ppy.sh) **from the email address attached to your account** to ask for details on your restriction. Make sure you include your username, along with any details you can provide as to why you think you may have been restricted. The more you tell us, the easier it is for us to resolve the issue.

> - If you strongly believe that your restriction is erreoneous, please contact us immediately so that we can discuss your case.

Reviewer: ```suggestion
- If you strongly believe that your restriction is erroneous, please contact us immediately so that we can discuss your case.
```

- **You cheated.** Using a third-party program, macros, or any form of unfair advantage will result in an immediate restriction.
- **Other cases.** The accounts team reserves the right to restrict an account for cases that are not covered by the three above.

> If an account has been found to be rightfully restricted, we will request the user to wait some time before sending their appeal. The duration of this cooldown varies depending on offences.

Reviewer: ```suggestion
If an account has been found to be rightfully restricted, we will ask that the user waits for some time before sending their appeal.  This is referred to as a *cooldown*, and its duration varies depending on the offences that have taken place.
```

- **Make sure you respect the rules.** Long-term restrictions are punishments for breaking the rules. If you continue to do so while restricted, the cooldown before you can appeal will be reset to three months from your last offence. This includes, for example, continuing to cheat, accessing others' accounts, or creating new accounts to evade a restriction.
- **Follow the support team's instructions.** The three-month cooldown is a chance to take time off and reflect on your actions. If you attempt submitting an appeal before the cooldown period is over, your emails will be ignored.  If you are unsure when your cooldown expires, when you will be allowed to appeal, or which of your accounts you should be playing on (and appealing from), you are welcome to send us an email to ask.
- **Do not hesitate to contact us if you have any questions.** As long as you are not trying to circumvent punishment, we will happily answer any questions you have about your account's restriction!

> ## My cooldown has passed. How do I appeal?

Reviewer 2: Could be 
```suggestion
## My cooldown period has passed. How do I appeal?
```

- **How you obtained the means to do so.** This applies mostly to cheating - we want to know where you obtained the tool you used, how you came across it, and whether or not it is still present on your computer.
- **Why we should allow you back.** Most of the time, this ties in to the second point. We just want to ensure that you will not be breaking the rules again! Please understand that the more rule violations in your account's history, the more we will expect in this section of the appeal.

## What if my appeal is granted?

If we decide to let you return, we will require that you agree to some terms before unrestricting your account.

- **You will not offend again.** This includes cheating and account sharing.
- **You will not create another account, be it for yourself or someone else.**
- **You should not associate with other accounts** including playing on shared computers.
- **Any major misbehaviour will result in immediate restriction**. This includes major silences.

Additionally, for a period of one year, we will not grant any flag changes for your account.

> In some cases, an appeal returnee may be banned from official or badged tournaments. This ban lasts one year and cannot be discussed. Some offences lead to a permanent ban from tournaments and higher positions.

Reviewer 3: both "or" and "and" read fine here (and mean the same thing... yay english), but i agree "and" feels better.

> In some cases, an appeal returnee may be banned from official and badged tournaments. This ban lasts one year and cannot be discussed. Some offences lead to a permanent ban from tournaments and higher positions.

Reviewer 4: ```suggestion
In some cases, an appeal returnee may be banned from official and badged tournaments. Including participate in as both player and stuff. This ban lasts one year and cannot be discussed. Some offences lead to a permanent ban from tournaments and higher positions.
```

Reviewer 4: might have grammar error.

Author: Good point. I'll add something along those lines.

> Some players may become restricted again once they start playing after having their appeal granted. This is an automated process and will generally be removed within 24 hours. Don't hesitate to poke us on twitter at [@osusupport](https://twitter.com/osusupport) or by sending an email to [accounts@ppy.sh](mailto:accounts@ppy.sh) if this happens to you!

Reviewer 2: I think "poke us" could be translated into a more formal manner, ie: 
```suggestion
Some players may become restricted again once they start playing after having their appeal granted. This is an automated process and will generally be removed within 24 hours. Don't hesitate to ask questions on twitter at [@osusupport](https://twitter.com/osusupport) or by sending an email to [accounts@ppy.sh](mailto:accounts@ppy.sh) if this happens to you!
```

- **Your history is too severe.** In cases of extreme offenders, or players who get restricted again after they have already been unrestricted on a successful appeal, we might simply state that your history is too severe to rejoin the game at the current time. Should this apply to you, we recommend waiting **a long time** before appealing again, and making sure you do not break any further rules in that time.

## Common restriction reasons and cooldowns

|Restriction reason|Cooldown duration|Appeal notes|
|-|-|-|
|Judgement Error|No cooldown|If found erroneous, no impact on the account.|
|Multi-account|Permanent (multis)|Main account will be left unrestricted, possibly silenced.|
|Excessive multi-accounting|3 months|Appeal terms apply|
|Account sharing|3 months|Appeal terms apply and partial rollback|
|Cheating|3 months|Appeal terms apply and full rollback|
|Excessive misconduct|Variable|Appeal terms apply. Cooldown will depend on history.|
|Cheating/Sharing for a tournament|6 months| Appeal terms apply, permanent tournament ban|

> |Second offences|6 months|Appeal terms apply, permanent tournament ban|

Reviewer 5: What about third offences and onwards?, is there something to note on those cases?

Reviewer 3: Probably change Second offences to Subsequent offences.
